src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -623,6 +623,12 @@ RECT NonClientIslandWindow::GetMaxWindowRectInPixels(const RECT* const prcSugges
             break;
         }
     }
+    case WM_DPICHANGED:
+    {
+        auto lprcNewScale = reinterpret_cast<RECT*>(lParam);
+        OnSize(RECT_WIDTH(lprcNewScale), RECT_HEIGHT(lprcNewScale));

```

Comment:
should basewindow call this?

Reply:
@DHowett-MSFT I honestly don't know. IslandWindow seems fine right now as it is. Do you want this for 0.3? If so, I won't have time to spin builds to verify before EOD. I'm fine making a follow up to investigate doing this. Thoughts?

Reply:
this fix is fine